cfgov/cfgov/settings/base.py

```diff
@@ -108,6 +110,32 @@
     "crtool",
 )
 
+WAGTAILSEARCH_BACKENDS = {
+    # The default search backend for Wagtail is the db backend, which does not
+    # support the custom search_fields defined on Page model descendents when
+    # using `Page.objects.search()`.
+    #
+    # Other backends *do* support those custom search_fields, so for now to
+    # preserve the current behavior of /admin/pages/search (which calls
+    # `Page.objects.search()`), the default backend will remain `db`.
+    #
+    # This also preserves the current behavior of our external link search,
+    # /admin/external-links/, which calls each page model's `objects.search()`
+    # explicitly to get results, but which returns fewer results with the
+    # Postgres full text backend.
+    #
+    # An upcoming effort to overhaul search within consumerfinance.gov and
+    # Wagtail should address these issues. In the meantime, Postgres full text
+    # search with the custom search_fields defined on our models is available
+    # with the "fulltext" backend defined below.
+    'default': {
+        'BACKEND': 'wagtail.search.backends.db',
+    },
+    'fulltext': {
+        'BACKEND': 'wagtail.contrib.postgres_search.backend',
+    },
```

Comment:
There's some additional discussion in the PR that added the external link search, https://github.com/cfpb/cfgov-refresh/pull/4540, about this.